### DISCUSSION TITLE
Frigate 0.14.1 doesnt work in HAOS

### DISCUSSION THREAD

**Discussion by jkbkster** [blakeblackshear/frigate#14132] [General Support]
My Frigate 0.14.1-f4f3cfa has stopped working after upgrade to 0.14xxx in HAOS.
Errorlog is one continous stream of errors.

Worked perfectly before. Cameras all cabled,  plenty of memory, disc, CPU and a coral TPU.

How to downgrade to eg 0.13 - or is there a fix ?

![Skærmbillede 2024-10-03 102403](https://github.com/user-attachments/assets/20ff2c04-1b3b-4534-a2a4-6985cfbde1bb)
![Skærmbillede 2024-10-03 102643](https://github.com/user-attachments/assets/1c99f4e6-4830-4aca-aa74-d094c03dedac)

**Comment by hawkeye217** [comment]
What is your frigate config? What hardware are you running? The error log is showing that your hardware acceleration device is unavailable.
